### DISCUSSION TITLE
Is there a way to refresh content on the web client without reloading the page?

### DISCUSSION THREAD

**Discussion by SteveDinn** [jellyfin/jellyfin#12712] [General]
Clicking on the "Home" link in the left hand menu used to do this, but somewhere along the way, it stopped doing this. I get why they would have stopped that behaviour from happening every time, when they could cache the content, but I still want a way to refresh it somehow.

I can still get it to happen by clicking in some nav link in the left-hand menu, and then back to home, but this seems needlessly cumbersome.

Maybe we could have a link on the Jellyfin logo that would refresh the content on the page, but not reload everything?

Thoughts?

**Comment by TheMelmacian** [comment]
Afaik there is no "update content" function implemented on the web client. The fastest way to trigger a page refresh is to use F5 or (STRG+F5) in your browser.
see this discussion: https://github.com/jellyfin/jellyfin/discussions/10700

**Comment by SteveDinn** [comment]
That's a shame. A hard refresh also resets any "Play On" device that might have been connected. I use Jellyfin-MPV-Shim to play content locally in order to prevent things being transcoded because they're being played in the browser.
